torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -2706,10 +2709,17 @@ def _register_post_forward_hooks(self):
         self._post_forward_handles.clear()
         for module in self.module.modules():
             module_param_handles = self._module_to_handles[module]
+            # reshard following handles: 1. the handle is in
+            # ``module_param_handles``and 2. the handle doesn't contain
+            # parameters in the parent module of the current module.
+            reshard_handles = []
```

Comment:
nit: Do you think this is easier to read?
```suggestion
            handles_to_reshard = []
```